### PR TITLE
Add new "SafeMedia" tier between Safe and PotentiallyUnwanted

### PR DESCRIPTION
**Pull request by jvyden**
This PR introduces a new tier for asset safety levels named "SafeMedia". It's intended to represent assets that are images or audio to help administrators single out images during content blocking.

Hopefully, this should replace our need to disable asset uploads on production.

### CODE REVIEW

File: Refresh.GameServer/Types/Assets/AssetSafetyLevel.cs [hunk 1]
```diff
@@ -5,15 +5,20 @@ public enum AssetSafetyLevel
     /// <summary>
     /// This asset is used in normal gameplay/operation and is okay to be uploaded.
     /// </summary>
+    /// <seealso cref="SafeMedia"/>
     Safe = 0,
     /// <summary>
+    /// This asset is still used in normal gameplay similar to <see cref="Safe"/>, and is okay to be uploaded, but the distinction can be useful.
```
Comment (Beyley) [inline Refresh.GameServer/Types/Assets/AssetSafetyLevel.cs:11]: this comment is hard to follow and doesnt actually explain what it means by media

**Review by Beyley — CHANGES_REQUESTED** [review]

File: Refresh.GameServer/Types/Assets/AssetSafetyLevel.cs (hunk 1 above)
Comment (jvyden) [reply Refresh.GameServer/Types/Assets/AssetSafetyLevel.cs:11]: I got ChatGPT to do it because I'm tired

**Comment by jvyden** [comment]
Not approved but don't care
